bin/email_sheets.py

```diff
@@ -1,4 +1,11 @@
-#!/usr/bin/env python
+# !/usr/bin/env python
```

Comment:
Minor typo, there is an extra space tucked away in here that should be removed.

```suggestion
#!/usr/bin/env python
```